src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -19,7 +21,23 @@ const buildIncludeUrl = (href, type) => {
 
   const withTrailingHref = href.startsWith('/') ? href : `/${href}`;
 
-  return `${process.env.SIMORGH_INCLUDES_BASE_URL}${withTrailingHref}${resolvers[type]}`;
+  const includeUrl = `${process.env.SIMORGH_INCLUDES_BASE_URL}${withTrailingHref}${resolvers[type]}`;
+
+  const currentRendererEnv = new Url(pathname, true).query.renderer_env;
+
+  let includeUrlWithParam = '';
+
+  switch (currentRendererEnv) {
+    case 'test':
+      includeUrlWithParam = addOverrideQuery(includeUrl, 'test');
+      break;
+    case 'live':
+      includeUrlWithParam = addOverrideQuery(includeUrl, 'live');
```

Comment:
could you just return in these two case statements and get rid of `includeUrlWithParam`?